pkg/types/azure/platform.go

```diff
@@ -0,0 +1,30 @@
+package azure
+
+import "strings"
+
+// Platform stores all the global configuration that all machinesets
+// use.
+type Platform struct {
+	// Region specifies the Azure region where the cluster will be created.
+	Region string `json:"region"`
+
+	// BaseDomainResourceGroupName specifies the resource group where the azure DNS zone for the base domain is found
+	BaseDomainResourceGroupName string `json:"BaseDomainResourceGroupName,omitempty"`
+
+	// UserTags specifies additional tags for Azure resources created for the cluster.
+	// +optional
+	UserTags map[string]string `json:"userTags,omitempty"`
```

Comment:
I don't think this is required..  right?

Reply:
well I don't know, I assumed it would be used internally in openshift.
In any case, it's good to be able to provide additional tags when deploying resources to Azure. Many company (including us internally) rely on resource tags to automatically clean or maintain dev subscriptions.